Multisig/UI/App/UIKit/UI Library/ReusableViews/AddressInfoView.swift

```diff
@@ -30,35 +41,31 @@ class AddressWithTitleView: UINibView {
         addTarget(self, action: #selector(didTouchUp(sender:forEvent:)), for: [.touchUpInside, .touchUpOutside, .touchCancel])
     }
 
-    func setStyle(_ value: Style) {
-        style = value
-        update()
-    }
-
-    func setAddress(_ value: Address) {
-        address = value
-        identiconView.setAddress(value.hexadecimal)
+    func setAddressInfo(_ addressInfo: AddressInfo) {
```

Comment:
I think it would be good to have a helper method that could assign address and label right away - from address or string

Reply:
Let's add it in future if there is such need. I have a feeling that for consistency and readability it is better always to operate with view model struct `AddressInfo`